Source/WebCore/platform/gtk/SelectionData.cpp

```diff
@@ -25,11 +25,11 @@
 
 namespace WebCore {
 
-static void replaceNonBreakingSpaceWithSpace(String& str)
+static void replaceNonBreakingSpaceWithSpace(String& string)
 {
-    static const UChar NonBreakingSpaceCharacter = 0xA0;
-    static const UChar SpaceCharacter = ' ';
-    str.replace(NonBreakingSpaceCharacter, SpaceCharacter);
+    static constexpr UChar NonBreakingSpaceCharacter = 0xA0;
+    static constexpr UChar SpaceCharacter = ' ';
```

Comment:
These both already exist in `CharacterNames.h`, `noBreakSpace` and `space`; we don’t need new ones here just for GTK.